README.md

```diff
@@ -255,11 +255,21 @@ There are a few different RDD's all named `Neo4jXxxRDD`
 
 * `Neo4jDataFrame`, a SparkSQL `DataFrame` that you construct either with explicit type information about result names and types
 * or inferred from the first result-row
-* Neo4jDataFrame provides `mergeEdgeList(sc: SparkContext, dataFrame: DataFrame, source: (label,Seq[prop]), relationship: (type,Seq[prop]), target: (label,Seq[prop]))` to merge a DataFrame back into a Neo4j graph
-  * both nodes are merged by first propery in sequence, all the others will be set on the entity, relat
-  * relationships are merged between the two nodes and all properties from sequence will be set on the relationship
-  * property names from the sequence are used as column names for the data-frame, currently there is no name translation
-  * the result are sent in batches of 10000 to the graph 
+* Neo4jDataFrame provides:
+ 1.`mergeEdgeList(sc: SparkContext, dataFrame: DataFrame, source: (label,Seq[prop]), relationship: (type,Seq[prop]), target: (label,Seq[prop]))` to merge a DataFrame back into a Neo4j graph
+    * both nodes are merged by first property in sequence, all the others will be set on the entity
+    * relationships are merged between the two nodes and all properties from sequence will be set on the relationship
+    * property names from the sequence are used as column names for the data-frame, currently there is no name translation
+    * the result are sent in batches of 10000 to the graph
+ 2. `mergeEdgeList(sc: SparkContext, dataFrame: DataFrame, source: (String,Seq[String]), relationship: (String,Seq[String]), target: (String,Seq[String]), renameMap: Map[String,String])` to do the
+ same as the first method except for `renameMap` parameter.<br/>
```

Comment:
can't we just add this one parameter to the first one with a default value (of an empty map).

Just curious why wouldn't you do that on the DF that gets passed in?

Reply:
@jexp thanks for reviewing this PR.

Agree, I removed the 2nd method and added a `renamedColumns` parameter to the original method (giving it a default value).

We can't do that in the DF that gets passed is as DF columns must have a unique name.
For example, if want to create ```(a:City)-[:HAS_ROAD_TO]->(b:City)``` relationship we need 2 columns named City